### ISSUE TITLE
Localization and browserify

### ISSUE THREAD

**Issue by tuelsch** [kylestetz/CLNDR#165]
Is it possible that clndr cannot be localized if used in a browserify setting because clndr is requiring its own instance of moment.js which can not be configured before initialization?

See http://stackoverflow.com/questions/29650119/localize-clndr-in-a-browserify-bundle for further description of the issue.

**Comment by tuelsch** [comment]
I forked the project and created a workaround for my problem. It's a one liner and if you are interested, I can create a pull request.

```
  // ...

  // The actual plugin constructor
  function Clndr( element, options ) {
      this.element = element;

    // If a moment object is provided in the options,
    // use this instead of the default one injected by UMD
    if (options.moment) moment = options.moment;

    // ...
```

**Comment by mikegioia** [comment]
This may be merged with #182 but I'll check it out further once I get browserify running with CLNDR and it's dependencies. I think there's a way to do it without needing the `options.moment` but I'll report back.

**Comment by tuelsch** [comment]
CLNDR could be shimmed so it uses the global scope references, but if jQuery and moment both are being bundled, there is no need to add jQuery and moment to the global scope, which would be nice.

**Comment by mikegioia** [comment]
@tuelsch I just added moment as a configuration option. I can't think of an easier solution. I have it here as a separate branch but I'd like to merge this into master soon on approval -- https://github.com/kylestetz/CLNDR/compare/moment_option

**Comment by kylestetz** [comment]
It seems like this will solve the immediate bug— changing the language configuration— but not #182, which is about clndr running `require` when used in a browserify environment (and not being able to stop that).

'm down with this change, though, since it's BC and it solves your issue!

**Comment by mikegioia** [comment]
Ah ok, I see now. I didn't know that require() calls were happening in clndr.js. I'll merge this in later today.

**Comment by mikegioia** [comment]
This is now merged into v1.3.2. I'm going to close this ticket out as #182 is open and concerns with the deeper issue.
